packages/math/init.lua

```diff
@@ -42,9 +42,13 @@ function package.declareSettings (_)
    })
    SILE.settings:declare({
       parameter = "math.font.size",
-      type = "integer",
-      default = 10,
+      type = "number or integer",
+      default = SILE.settings:get("font.size"),
    })
+   SILE.settings:registerHook ("font.size", function (size)
+      -- Follow document font size changes
+      SILE.settings:set("math.font.size", size)
```

Comment:
If we do allow a relative value (e.g. `1.2em`) then this would need to adjust to compute the difference (or just not change it at all, but we would have to review if we are using `:absolute()` where necessary) not just hard code a value.

Even if not, we probably want to figure out how to *remove* this hook if the user resets `font.size` after having set `math.font.size` at any point.

Reply:
The more I think about it, the more I tend to believe that `math.font.size` should be killed and we use just the current font size.
Let me know you feeling (whether we address it in 0.15.8, whether we don't, and what we foresee) -- So I can document it (or not) in an upcoming booklet.

Reply:
> The more I think about it, the more I tend to believe that `math.font.size` should be killed and we use just the current font size.

I don't see how that would work. We're automatically switching font faces for Math and it may or may not be from the same font family or appropriately sized to be matched with the body text. I don't see how we can get away with *not* allowing separate control over the size when we are changing the face.

> Let me know you feeling (whether we address it in 0.15.8, whether we don't, and what we foresee) -- So I can document it (or not) in an upcoming booklet.

I'm happy to move this forward as soon as we agree on how it *should* be handled. I just don't want to push out a change that we know has just a different set of downside to the current one. If perchance the solution we come up with ends up being a significant breaking change that might make it more complicated and delay things as well, but as long as this is a setting issue we can probably shim it in the *retrograde* package and not worry about it some reflows being breaking.

Reply:
Noted!
I removed my assignment and shifted the original issue to 0.x.y -- It might be reconsidered indeed, but that won't occur as part of my on-going 2024 effort on math which is soon reaching its end.